srcpkgs/riot-desktop/template

```diff
@@ -12,7 +12,7 @@ maintainer="projectmoon <projectmoon@agnos.is>"
 license="Apache-2.0"
 homepage="https://riot.im"
 distfiles="https://github.com/vector-im/riot-desktop/archive/v${version}.tar.gz>riot-desktop.tar.gz https://github.com/vector-im/riot-web/archive/v${version}.tar.gz>riot-web.tar.gz"
-checksum="4b1befbabb1fc391d0a9a49125540903fa97d5bbad033bdaff377a018a244333 0e54adbda71e2d572f1ecdec3a6f07cb6379e5c9f8223bff19580d2e663f50c4"
```

Comment:
break this into two lines, second line indented with a space

    checksum="4b1befbabb1fc391d0a9a49125540903fa97d5bbad033bdaff377a018a244333
     0e54adbda71e2d572f1ecdec3a6f07cb6379e5c9f8223bff19580d2e663f50c4"